chainer/backends/_chainerx.py

```diff
@@ -17,9 +17,7 @@ class ChainerxDevice(_backend.Device):
 
     __hash__ = _backend.Device.__hash__
 
-    def __init__(self, device):
-        # type: (chainerx.Device) -> None
-
+    def __init__(self, device: 'chainerx.Device') -> None:
```

Comment:
69a9ba2

I annotated it using string because I got the folowing error.
`[pylint E1101] [E] Module 'chainerx' has no 'Device' member (no-member)`